### DISCUSSION TITLE
Zmiana treści błędu na bardziej zrozumiały podczas wystawiania ofert produktowych

### DISCUSSION THREAD

**Discussion by jezewskiBNetrox** [allegro/allegro-api#8036] [Suggestions - resolved]
Witam,
mała sugestia bo sam ślęczałem chwilę żeby zrozumieć.

Błąd:
```
Podany identyfikator wartości słownikowej `3786_85` w parametrze o identyfikatorze`3786` wskazuje na inną wartość niż przekazana w polu `values`.
```

Jest kompletnie niezrozumiały. W moim przypadku problem polegał na tym, że marka podana jako inna i wpisana ręcznie jest na liście słownikowej i ją trzeba wybrać. W starym zasobie POST /sale/offers ten błąd był jasny, bo jasno określał że próbuje się wpisać w "inna" wartość znaną na liście. 

Proszę o rozróżnienie tego, bo w tej formie ciężko zrozumieć co tak naprawdę jest problemem.

**Comment by MartaNowaczyk** [comment]
Pracujemy nad dodaniem nazw dla tych wartości z komunikatu, aby był bardziej czytelny. Nie mamy jednak w planach przekazywać wartości, którą dodano jako niejednoznaczną - czyli wprowadzoną ręcznie.

**Reply by jezewskiBNetrox** [reply]
@MartaNowaczyk Chyba się nie zrozumieliśmy. Komunikat powinien przekazywać, że problemem jest wpisanie jako wartości niejednoznacznej takiej, która już istnieje w słowniku. Zamiast tego jest jakiś niezrozumiały bełkot, że w values wpisano inną wartość niż "inna".. czyli wpisana ręcznie? Komunikat jest nieczytelny nie z powodu zastosowania identyfikatorów parametrów.

**Reply by MartaNowaczyk** [reply]
Takie zapotrzebowanie też już wcześniej zgłosiłam.
